documentation/docs/release-notes/2.44.0.1.md

```diff
@@ -0,0 +1,75 @@
+# Percona Monitoring and Management 2.44.0-1
+
+**Release date**: February 10th, 2025                                                                            
+
+Percona Monitoring and Management (PMM) is an open-source database monitoring, management, and observability solution for MySQL, PostgreSQL, and MongoDB. 
+
+It provides tools to observe database health, analyze performance trends, troubleshoot issues, and execute database management tasks—whether your databases are on-premises or in the cloud.
+
+
+## Security advisory: Critical vulnerability
+
+A critical vulnerability has been identified in OVA installations of PMM 2.38 and above, as described in [CVE-xxxx-xxxx](https://cve.mitre.org/cgi-bin/cvename.cgi?name=xxxx-xxxx). If you are running any of these OVA installations, we strongly recommend upgrading immediately to this release.
+
+The vulnerability was caused by default service account credentials used in OVA provisioning, which had weak password security and enabled SSH login access. 
+These issues could allow unauthorized SSH access and potential privilege escalation to `root` via `sudo` capabilities.
```

Comment:
and leak credentials of all services added to PMM. So we recommend to rotate passwords on these services as well